lib/dynflow/execution_plan/steps/abstract.rb

```diff
@@ -128,11 +134,15 @@ def with_sub_plans?
         false
       end
 
+      def queue
```

Comment:
This class already has attribute reader for `queue`. This could be dropped

Reply:
Right, deleted.